src/utils/Processing.ts

```diff
@@ -240,6 +241,26 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+function checkNullValuesOnArray(resource: any): void {
+  const resourceType = resource instanceof InstanceDefinition ? 'Instance' : resource.resourceType;
+  const resourceName =
+    resource instanceof InstanceDefinition ? resource._instanceMeta.name : resource.title;
+  for (const property in resource) {
+    if (!property.startsWith('_') && Array.isArray(resource[property])) {
+      const nullIndexes: number[] = [];
+      resource[property].forEach((element: any, index: number) => {
+        if (element === null) nullIndexes.push(index);
+      });
+      if (nullIndexes.length > 0)
+        logger.warn(
+          `Null values found at elements ${JSON.stringify(
+            nullIndexes
+          )} within '${property}' on ${resourceType} '${resourceName}'`
```

Comment:
IIRC, `JSON.stringify` on an array will include the `[` / `]` delimiters.  So this message will end up something like:
> Null values found at elements [2,3] within 'foo' on StructureDefinition 'MyProfile'

I think people might find that confusing, so I might change that to read something more like:

> The array 'foo' in MyProfile is missing values at the following indices: 2, 3.